File: app/src/main/java/com/candy/capture/util/TimeUtil.java
package com.candy.capture.util;

/**
 * Created by zhanghq on 2016/11/10.
 */

public class TimeUtil {
    /**
     *
     * @param duration 秒
     * @return
     */
    public static String formatDuration(long duration) {
        String time = String.format("%02d:%02d", duration / 60, duration % 60);
        return time;
    }
}

File: app/src/main/java/com/candy/capture/util/TimeUtil.kt
package com.candy.capture.util

/**
 * @Description
 * @Author zhanghaiqiang
 * @Date 2020/9/30 15:47
 */
object TimeUtil {
    /**
     *
     * @param duration 秒
     * @return
     */
    fun formatDuration(duration: Long): String? {
        return String.format("%02d:%02d", duration / 60, duration % 60)
    }
}